src/main.cpp

```diff
@@ -192,7 +192,7 @@ void DDWAF::toggleRules(const Napi::CallbackInfo& info) {
   }
 
   ddwaf_object rulesToggleMap;
-  to_ddwaf_object(&rulesToggleMap, env, info[0], 0, false, false);
+  to_ddwaf_object(&rulesToggleMap, env, info[0], 0, false);
```

Comment:
we should make this var false by default no ? so we don't need to specify it in the other calls.

Reply:
`coerceToString` is false by default - check [here](https://github.com/DataDog/dd-native-appsec-js/pull/71/files#diff-60230de8992995e443be0b45a6db16dd250757c027b9e16b7946e9ff4d68c9a9R19)

Reply:
Ooooh whoops I read it wrong, I thought you added the argument `false` here, but you actually removed it my bad